api/v1/publicationPeerReviews/resources/PublicationPeerReviewResource.php

```diff
@@ -111,11 +111,13 @@ private function getReviewAssignmentPeerReviews(Enumerable $assignments, Context
             } else {
                 $reviewerComments = $this->getReviewAssignmentComments($assignment);
             }
+
+            $isReviewOpen = $assignment->getReviewMethod() === ReviewAssignment::SUBMISSION_REVIEW_METHOD_OPEN;
```

Comment:
I wonder if we'd like to include this in the data that's returned. I know it will be obvious in context, but it may be helpful to have this codified in the data returned. Something like `'isReviewOpen => true`.

Reply:
Good idea -- I've added it to the data returned.